src/main/java/org/betonquest/betonquest/compatibility/Compatibility.java

```diff
@@ -154,22 +155,32 @@ private void hook(final Plugin hook) {
         }
 
         // hook into the plugin if it's enabled in the config
-        if ("true".equalsIgnoreCase(plugin.getConfig().getString("hook." + name.toLowerCase(Locale.ROOT)))) {
+        if ("true".equalsIgnoreCase(betonQuest.getConfig().getString("hook." + name.toLowerCase(Locale.ROOT)))) {
             LOG.info(null, "Hooking into " + name);
 
             // log important information in case of an error
             try {
                 integrator.hook();
                 hooked.add(name);
             } catch (final HookException exception) {
-                final String message = String.format("There was an error while hooking into %s %s (BetonQuest %s, Spigot %s)! %s",
-                        exception.getPluginName(),
-                        exception.getPluginVersion(),
+                final String message = String.format("Could not hook into %s %s! %s",
+                        hookedPlugin.getName(),
+                        hookedPlugin.getDescription().getVersion(),
+                        exception.getMessage());
+                LOG.warning(null, message);
+                LOG.debug(null, message, exception);
```

Comment:
```suggestion
                LOG.warning(null, message, exception);
```
The `warning` method already puts the exception in the debug log and does not print it out. To actually print the stack trace to the server log you would have to use the `error` method (we don't want that here).